qiita_ware/test/test_ebi.py

```diff
@@ -398,6 +420,26 @@ def test_generate_demultiplexed_fastq(self):
         self.assertItemsEqual(obs_demux_samples,
                               ebi_submission.samples_prep.keys())
 
+    def test_generate_send_sequences_cmd(self):
+        ppd = self.write_demux_files(PrepTemplate(1))
+        e = EBISubmission(ppd.id, 'ADD')
+        e.generate_demultiplexed_fastq()
+        self.files_to_remove.append(e.ebi_dir)
+        e.write_xml_file(e.generate_study_xml(), e.study_xml_fp)
+        e.write_xml_file(e.generate_sample_xml(), e.sample_xml_fp)
+        e.write_xml_file(e.generate_experiment_xml(), e.experiment_xml_fp)
+        e.write_xml_file(e.generate_run_xml(), e.run_xml_fp)
+        e.write_xml_file(e.generate_submission_xml(), e.submission_xml_fp)
+        obs = e.generate_send_sequences_cmd()
+
+        exp = ('ascp --ignore-host-key -L- -d -QT -k2 '
```

Comment:
Well, I'm not sure how else to test. The method basically builds a command based on the available/successful fastq.gz files. In other words, what do you suggest testing? The expansion? Well, that's tested somewhere else where we validate that only good sequence files are generated.